README.markdown

```diff
@@ -257,142 +356,40 @@ Specifies whether the subsetting should be present. Valid options: 'present' and
 
 *Optional.* Supplies a value for the specified subsetting. Valid options: a string. Default value: undefined.
 
-
-
-
 ### Function: create_ini_settings
 
-`create_ini_settings($settings, $defaults)`
+Manages multiple `ini_setting` resources from a hash. Note that this cannot be used with ini_subsettings.
 
-Manage multiple ini_setting resources from a hash with comfort. You can provide a hash in your manifest and feed it from Hiera. This can however not be used with ini_subsettings!
+`create_ini_settings($settings, $defaults)`
 
-#### Parameters
+#### Arguments
 
-##### `$settings`
+##### First argument: `settings`
 
-*Required.* Specify a hash with the ini_setting resources.
+*Required.* Specify a hash representing the `ini_setting` resources you want to create.
 
-###### Example
-~~~
-defaults = { 'path' => '/tmp/foo.ini' }
-$example = { 'section1' => { 'setting1' => 'value1' } }
-create_ini_settings($example, $defaults)
-~~~
-results in a resource
-~~~
-ini_setting { '[section1] setting1':
-    ensure  => present,
-    section => 'section1',
-    setting => 'setting1',
-    value   => 'value1',
-    path    => '/tmp/foo.ini'
-}
-~~~
+##### `defaults`
 
-###### Example with special parameters
-~~~
-defaults = { 'path' => '/tmp/foo.ini' }
-$example = { 'section1' => {
-                'setting1' => 'value1',
-                'settings2' => {
-                        'ensure' => 'absent'
-                    }
-                }
-           }
-create_ini_settings($example, $defaults)
-~~~
-results in resources
-~~~
-ini_setting { '[section1] setting1':
-    ensure  => present,
-    section => 'section1',
-    setting => 'setting1',
-    value   => 'value1',
-    path    => '/tmp/foo.ini',
-}
-ini_setting { '[section1] setting2':
-    ensure  => absent,
-    section => 'section1',
-    setting => 'setting2',
-    path    => '/tmp/foo.ini',
-}
+*Optional.* Accepts a hash to be used as the values for any attributes not defined in the first argument.
+ 
+~~~puppet
+`$example = {
+  'section1' => {
+    'setting1' => {
+      'value' => 'value1', 'path' => '/tmp/foo.ini' 
+      } 
+    } 
+  }`
```

Comment:
These three close curly braces should be UNindented 2 spaces (so they match the indentation of the lines of the corresponding open curlies).